src/plugins/intel_gpu/tests/test_cases/canonicalization_gpu_test.cpp

```diff
@@ -103,10 +103,10 @@ struct broadcast_params {
 };
 
 std::vector<std::pair<Shapes, broadcast_params>> broadcast_shapes_with_params {
-    {{{{5}}, {{1, 1, 5, 1}}, {{3, 1, 5, 1}}}, {{3, 1, 5}, {}, ov::op::BroadcastType::NUMPY}},
-    {{{{5}}, {{1, 1, 1, 1, 5}}, {{1, 2, 3, 4, 5}}}, {{1, 2, 3, 4, 5}, {}, ov::op::BroadcastType::NUMPY}},
-    {{{{3, 1}}, {{1, 1, 3, 1}}, {{1, 2, 3, 4}}}, {{1, 2, 3, 4}, {}, {ov::op::BroadcastType::PDPD, 2}}},
-    {{{{4, 1, 6}}, {{1, 1, 1, 4, 1, 6}}, {{1, 2, 3, 4, 5, 6}}}, {{1, 2, 3, 4, 5, 6}, {}, {ov::op::BroadcastType::PDPD, 3}}}
+    {{{{5}}, {{5, 1, 1, 1}}, {{3, 1, 5, 1}}}, {{3, 1, 5}, {}, ov::op::BroadcastType::NUMPY}},
+    {{{{5}}, {{5, 1, 1, 1, 1}}, {{1, 2, 3, 4, 5}}}, {{1, 2, 3, 4, 5}, {}, ov::op::BroadcastType::NUMPY}},
+    {{{{3, 1}}, {{3, 1, 1, 1}}, {{1, 2, 3, 4}}}, {{1, 2, 3, 4}, {}, {ov::op::BroadcastType::PDPD, 2}}},
+    {{{{4, 1, 6}}, {{4, 1, 6, 1, 1, 1}}, {{1, 2, 3, 4, 5, 6}}}, {{1, 2, 3, 4, 5, 6}, {}, {ov::op::BroadcastType::PDPD, 3}}}
```

Comment:
@Lyamin-Roman 
After apply my changes, canonicalization.broadcast unit test occurred failed, so I changed to fix it. Is it acceptable as your intention?